server/controllers/answers.js

```diff
@@ -1,6 +1,6 @@
-const {validationResult} = require('express-validator');
-const helperFunction = require('../helpers/helperFunction');
-const Answer = require('../models/answers.model');
+import validator from 'express-validator';
```

Comment:
Here, `import { validationResult } from 'express-validator';` would be better as only one function is being used from `express-validator` in this file.

Same thing can be changed in 2-3 more places.

Reply:
Yes! It's would be better! You right! But, for it be possible I need update the 'express-validator', because the version that is in the project don't accept destruction of individual functions.

Reply:
Yes, you can but after doing that just check if there are any new errors coming up.

Reply:
Hi, have you made these changes?

Reply:
No, yet...

Reply:
When I do, you will can see the changes and I'll comment here...

Reply:
Its is in my kaban, dont worry @Mayank0255

Reply:
Oh glad to hear that👍

Reply:
@Mayank0255 this plugin 'express-validator' isn't updated for be possible do destructuring in the line of import. When this dependency update is available we can do it.

**FYI, the reason about:** https://stackoverflow.com/questions/46927589/express-validator-usage-with-esm-modules

But, I did destructing with the variable of the export. For example:

import validator from 'express-validator'
const {validationResult} = validator;

This is the best it's possible now.

Reply:
Cool, works for now!